receiver/azureeventhubreceiver/eventhubhandler_test.go

```diff
@@ -22,11 +22,14 @@ import (
 	eventhub "github.com/Azure/azure-event-hubs-go/v3"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
+
 	"go.opentelemetry.io/collector/component"
 	"go.opentelemetry.io/collector/component/componenttest"
+	"go.opentelemetry.io/collector/consumer"
 	"go.opentelemetry.io/collector/consumer/consumertest"
 	"go.opentelemetry.io/collector/obsreport"
 	"go.opentelemetry.io/collector/receiver/receivertest"
+	"go.uber.org/zap"
```

Comment:
you need to run goimports and will need to move this import into the group up.

Reply:
Looking at the go.mod it appears that zap was available.  Is there something I'm missing?

Reply:
it's just the way the imports are set. Try to run `make lint` or `make impi` in the folder to see the errors.